### PR TITLE
ErrataConnection: add raw kwargs to _get method

### PR DESCRIPTION
**Pull request by yazug**
- added doctext to this method
- added support for 'raw' bool kwargs defaulting to False
- return the whole response object if raw set, otherwise
  return json() results like normal

This is to allow usage of _get() for retreiving other content than API
where the returned content is always json object.

User of this option is responsible for handling error checking and decoding

### CODE REVIEW

File: errata_tool/connector.py [hunk 1]
```diff
@@ -162,7 +175,11 @@ def _get(self, u, **kwargs):
 
         if ret_json is None and ret_data is not None:
             if ret_data.status_code == 200:
-                ret_json = ret_data.json()
+                if return_json_decoded_data:
+                    ret_json = ret_data.json()
+                else
```
[the remark this answers is not in the document]
Comment (yazug) [reply errata_tool/connector.py]: done

File: errata_tool/connector.py [hunk 2]
```diff
@@ -137,11 +137,22 @@ def _post(self, u, **kwargs):
         return ret
 
     def _get(self, u, **kwargs):
+        """_get is convience method that retrives content from server
+
+        Recognized kwargs
+            'data' for requests data object to send with get call
+            'json' for requests json object to send with get call
+            'raw' bool (defaulting to False) for returning whole response object
```
[the remark this answers is not in the document]
Comment (yazug) [reply errata_tool/connector.py]: done

**Comment by yazug** [comment]
yea, there is a slight discrepancy in _get vs _post in what is returned ... and without breaking api compatibility no good way to change that behaviour

**Comment by yazug** [comment]
not sure but likely would be good to get unittest around this functionality.

**Comment by ktdreyer** [comment]
Did you want to add tests or merge as-is?

**Comment by yazug** [comment]
I'm ok to merge as is and will look to add more unit tests around that functionality in a separate patch/pull request

**Pull request merged**